### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by assertchris**
Brute-force deterrent.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -692,6 +692,7 @@ try {
     $log->info($e->getMessage());
     throw new InvalidLoginException();
 } catch (InvalidLoginException $e) {
+    sleep(5);
```
Comment (harikt) [inline README.md:695]: Why sleep ?

**Comment by harikt** [comment]
I feel, there should be a field to add the number of failed attempts. After certain number of attempts we should lock the status.

File: README.md (hunk 1 above)
Comment (assertchris) [inline README.md:695]: Exponentially increases the amount of time a blind brute-force attack would take to work.

**Comment by assertchris** [comment]
I agree, the best would be a growing delay between attempts, after a certain threshold. That's not something easily done. Would make a nice addition to the library though...

**Comment by harikt** [comment]
In the case of PDO I assume we need some sort of table to be structured. Else the possibility of brute force will be high. Make use of captcha in login etc will help things .

**Comment by assertchris** [comment]
Any cache provider will do. I'm uninspired by captcha.

**Comment by harikt** [comment]
@assertchris cache provider ?

**Comment by assertchris** [comment]
In order to identify (especially in a distributed environment) repeated attempts from the same IP or username, it would be prudent to use semi-persistent storage. Something like https://github.com/doctrine/cache or https://github.com/illuminate/cache would provide the database aspects you were talking about, but also give the option for APC cache, file cache etc.

**Comment by harikt** [comment]
aah ok :) . But I don't think cache is not what we need here.

My mind may change if I find the usage good enough.

Thank you.

**Comment by pmjones** [comment]
While I get the point of delaying further login attempts, using `sleep()` like this doesn't seem like a good example to set.

**Pull request closed without merging**